packages/deployments/contracts/deploy/02_deployConnext.ts

```diff
@@ -305,13 +305,22 @@ const func: DeployFunction = async (hre: HardhatRuntimeEnvironment): Promise<voi
     });
     console.log("TestERC20: ", deployment.address);
 
+    deployment = await hre.deployments.deploy("TestAdopted", {
+      contract: "TestERC20",
+      from: deployer.address,
+      log: true,
+      skipIfAlreadyDeployed: true,
+      args: ["Test Adopted", "TEST2"],
```

Comment:
TEST2? Not just TEST?

Reply:
it'll make it easier to debug

Reply:
I just wanted to make new testing token have separate adopted and local assets.